src/Control/Monad/Aff.js

```diff
@@ -320,3 +320,11 @@ exports._tailRecM = function (isLeft, f, a) {
     }(a);
   };
 };
+
+
+var avar = require("../Control.Monad.Aff.AVar/foreign.js");
```

Comment:
I don't think `pulp test` uses browserify, and it succeeds there, so no, I think it's ok. I was concerned about that too... I'm still a little suspicious, but it seemed to work for the cases I tested. I'll try with `psc-bundle` directly to ensure there isn't some magic in `pulp`.